pkgs/applications/version-management/helix-cli/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, patchelf }:
+
+stdenv.mkDerivation rec {
+  name = "helix-cli-${version}";
+  version = "2018.1.1705517";
+
+  src = fetchurl {
+    url = "https://cdist2.perforce.com/perforce/r18.1/bin.linux26x86_64/p4";
+    sha256 = "770c38fbd3726bd11abfd450900d492b41b5bd6049703a063fca20d1a70ecdc6";
+  };
+
+  dontBuild = true;
+  phases = [ "installPhase" ];
```

Comment:
We want to deprecate explicitly setting phases.

```suggestion
```

This might cause an issue building so feel free to paste it.